packages/script.js

```diff
@@ -1,7 +1,7 @@
+import BASE_URL from '../lib/baseUrl';
```

Comment:
`parcel` is able to handle this import and inline the configured env variable, that's why I replaced `uglify-js` with it.

Reply:
Thanks!